include/fluent-bit/flb_pack.h

```diff
@@ -51,7 +51,7 @@
     "supported formats: double, " \
      "iso8601 (e.g: 2018-05-30T09:39:52.000681Z), " \
      "java_sql_timestamp (e.g: 2018-05-30 09:39:52.000681, useful for AWS Athena), "\
-     "and epoch."
+     "epoch and epoch_ms."
```

Comment:
I think it was changed to `epoch_milliseconds` in the PR?

Reply:
It's this:

```
else if (strcasecmp(str, "epoch_ms") == 0 ||
             strcasecmp(str, "epoch_millis") == 0 ||
             strcasecmp(str, "epoch_milliseconds") == 0) {
```

Reply:
Hi @PettitWesley,
From my understanding, the 3 of them ( `epoch_ms`, `epoch_millis` and `epoch_milliseconds`  ) are equally accepted.

But I think fluent-bit's help could just show one of them. I believe it may be more confusing otherwise.
What do you think ?

Reply:
I think the docs and help text need to clearly state all acceptable options- otherwise, how will users find out what works?

Reply:
Shouldn't we leave this to the big (HTML) documentation ? what should the text be like ? Won't it clutter too much the CLI ?

Reply:
@PettitWesley done !